apis/policies/v1/clusteradmissionpolicy_webhook.go

```diff
@@ -68,7 +68,8 @@ var _ webhook.Validator = &ClusterAdmissionPolicy{}
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type
 func (r *ClusterAdmissionPolicy) ValidateCreate() error {
 	clusteradmissionpolicylog.Info("validate create", "name", r.Name)
-	return nil
+
+	return validatePolicyCreate(r)
```

Comment:
we could just call `validateRulesField(policy)` directly.

Reply:
I was hesitant to do so in case new validations come into play but it makes sense.

Reply:
Updated!